hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSideWithCoprocessor.java

```diff
@@ -43,7 +43,7 @@
   @Parameterized.Parameters
   public static Collection parameters() {
     return Arrays.asList(new Object[][] {
-        { ZKConnectionRegistry.class, 1}
+        { MasterRegistry.class, 1}
```

Comment:
What you thinking? No tests of old registry in Master?

Reply:
I'd prefer to retain both. Let's follow on with some work to establish code coverage and see about pruning the tests via that angle.